gobblin-compaction/build.gradle

```diff
@@ -31,6 +31,7 @@ dependencies {
   // with hive-exec-core in older version(1.0.1), we need to shadow ord-mapreduce's transitive deps.
   // and include direct orc-mapreduce library just as a compileOnly dependency
   compileOnly externalDependency.orcMapreduce
+  compile externalDependency.orcTools
```

Comment:
Should it be `testCompile`?

Reply:
Since the utility class that allows users to run compaction locally is not a unit test, I had it located in the src/main of gobblin-compaction module. Also, this dependency will be needed at runtime, hence I chose "compile".

Reply:
oh OK I didn't notice that ( utility class in src instead of test).